virtualenvapi/manage.py

```diff
@@ -77,11 +77,6 @@ def root(self):
         """The root directory that this virtual environment exists in."""
         return os.path.split(self.path)[0]
 
-    @property
-    def name(self):
```

Comment:
virtualenv can handle absolute path of a target directory, let's use it instead of relative one.